azurerm/internal/services/cdn/parse/cdn_profile.go

```diff
@@ -32,3 +38,8 @@ func CdnProfileID(input string) (*CdnProfileId, error) {
 
 	return &profile, nil
 }
+
+func (id CdnProfileId) ID(subscription string) string {
```

Comment:
_minor_ to make this consistent with the other formatters can we make this `subscriptionId`:

```suggestion
func (id CdnProfileId) ID(subscriptionId string) string {
```